charts/logzio-k8s-events/templates/clusterrole.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: {{ .Values.apiVersions.clusterRole }}
 kind: ClusterRole
 metadata:
   name: {{ include "logzio-k8s-events.fullname" . }}
-  namespace: {{ .Values.namespace }}
+  namespace: {{ .Release.Namespace }} 
```

Comment:
I'm not sure if clusterrole requires a namespace, please check.

Reply:
It doesn't, removed.